app/views/categories/index.html.erb

```diff
@@ -1,14 +1,30 @@
 <p style="color: green"><%= notice %></p>
+<%= button_to "Log Out", destroy_user_session_path, class: "button is-link is-rounded is-normal is-danger", method: :delete, data:{turbo:false} %>
 
-<h1>Categories</h1>
-
-<div id="categories">
-  <% @categories.each do |category| %>
-    <%= render category %>
-    <p>
-      <%= link_to "Show this category", category %>
-    </p>
-  <% end %>
-</div>
-
-<%= link_to "New category", new_category_path %>
+  <h1 class="title is-3 has-text-centered mt-5">Categories</h1>
+  <div class="container">
+    <% @categories.each do |category| %>
+      <div class="block m-6 has-background-info-light p-5 ">
+        <div class="media-left">
+          <figure class="image is-64x64 margin-4">
+            <% if category.present? && category.image.attached? %>
+              <%= image_tag(category.image) %>
+            <% end %>
+          </figure>
+        </div>
+        <div class="media-content">
+          <strong>    <%= link_to category.name, category_entities_path(category) %>
+          </strong>
+        </div>
+        <p class="m-4">
+          <%= link_to "Show this category", category, class: "button is-danger is-rounded is-small
+" %>
+        </p>
+      </div>
+    <% end %>
+  </div>
+  <footer>
+    <div class="buttons is-centered">
+      <%= link_to "Add New Category", new_category_path, class: "button is-link is-light is-rounded is-normal" %>
+    </div>
+    <footer>
```

Comment:
- [ ] Kindly style your categories page as we have in the design. Kindly take note of the colors and composition of the elements on the page.

  Your Implementation | Behance Design
  :---:|:---:
  ![ieoewp](https://user-images.githubusercontent.com/54848363/214603228-6b296d70-114e-4324-9aa4-b294db1ea0a8.jpg) | ![riew](https://user-images.githubusercontent.com/54848363/214603237-798e428d-4e6f-49b2-b55d-01369dbce9a8.jpg)